### PR TITLE
Add Bar Chart to  AboutMe.html, Add data to drawChar in script.js

### PR DESCRIPTION
**Pull request by NzingaHolloman**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -58,22 +58,27 @@ google.charts.load('current', {'packages':['corechart']});
...
-  const data = new google.visualization.DataTable();
-  data.addColumn('string', 'Animal');
-  data.addColumn('number', 'Count');
-        data.addRows([
-          ['Brazil!', 10],
-          ['Mexico!', 5],
-          ['Alaska!', 15]
-        ]);
+var data = google.visualization.arrayToDataTable([
+        ['Destination', 'Votes',],
```
Comment (josueetcom) [inline portfolio/src/main/webapp/script.js:62]: Is the `,` before `]` supposed to be there?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -58,22 +58,27 @@ google.charts.load('current', {'packages':['corechart']});
...
+        ['Brazil', 10],
+        ['Mexico', 6],
+        ['Alaska', 8],
+        ['Texas', 11],
+        ['Philadelphia', 9]
```
Comment (josueetcom) [inline portfolio/src/main/webapp/script.js:67]: [Style](https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma): Include a trailing comma whenever there is a line break between the final element and the closing bracket.

```suggestion
        ['Philadelphia', 9],
```

This makes it easier to add more items without this particular line appearing changed.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -58,22 +58,27 @@ google.charts.load('current', {'packages':['corechart']});
...
+      var options = {
+        title: '',
+        chartArea: {width: '50%'},
+        hAxis: {
+          title: 'Votes',
+        },
+        vAxis: {
+          title: 'Destinations',
+          minValue: 20
+        }
```
Comment (josueetcom) [inline portfolio/src/main/webapp/script.js:78]: [Style](https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma): Include a trailing comma whenever there is a line break between the final property and the closing brace.

```suggestion
        },
```

For the same reason as above 😃

**Review by josueetcom — APPROVED** [review]
Just some style changes but otherwise LGTM! 🎉

**Review by josueetcom — CHANGES_REQUESTED** [review]
Changing to **Request changes** to make sure comments from PR #11 are addressed either there or here 😄

**Comment by TengkuHanaShafikah** [comment]
Other than Josue's comments above, everything looks good to me!

**Review by TengkuHanaShafikah — COMMENTED** [review]
Other than Josue's comments above, everything looks good to me! Nice job!

**Pull request merged**